Tests/scripts/utils/GoogleSecretManagerModule.py

```diff
@@ -16,17 +16,27 @@ def get_secret(self, project_id: str, secret_id: str, version_id: str = 'latest'
             logging.error(
                 f'Secret json is malformed for: {secret_id} version: {response.name.split("/")[-1]}, got error: {e}')
 
-    def list_secrets(self, project_id: str, name_filter: list = [], with_secret=False) -> list:
+    def list_secrets(self, project_id: str, name_filter: list = [], with_secret=False, attr_validation=[]) -> list:
```

Comment:
don't use mutables (`[]`) as default args. in this case, we can use a `tuple[str, ...]` and change the square brackets in `add_secrets_file_to_build` to round ones.